setup.py

```diff
@@ -48,7 +48,6 @@
         'Programming Language :: Python',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
```

Comment:
It'd be good to add these whilst you're at it, allows for getting better stats from PyPI:
```
'Programming Language :: Python :: 2',
'Programming Language :: Python :: 3',
```